apps/src/code-studio/initApp/loadApp.js

```diff
@@ -289,6 +289,10 @@ function loadAppAsync(appOptions) {
     return loadProjectAndCheckAbuse(appOptions);
   }
 
+  // shouldLoadChannel will be true for channel backed levels on publicly-cached pages
+  // We will need to load the channel client-side from api/user_progress
+  const shouldLoadChannel = appOptions.shouldLoadChannel && !appOptions.channel;
```

Comment:
(minor) I think we can omit the `!appOptions.channel` because the function would have exited at 289 if appOptions.channel were present.  Up to you.